<launch>
  <param name="use_sim_time" value="true"/>

  <group ns="detect_and_locate">
    <rosparam file="$(find depth_image_extractor)/config/object_detection_rock.yaml"/>
    <rosparam file="$(find depth_image_extractor)/config/pose_estimator_rock.yaml"/>
  </group>
  <node name="detect_and_locate" pkg="depth_image_extractor" type="detection_and_position_node" output="screen">
    <remap from="/camera/color/image_raw" to="/camera01/color/image_raw"/>
    <remap from="/camera/aligned_depth_to_color/image_raw" to="/camera01/aligned_depth_to_color/image_raw"/>
  </node>
</launch>